apps/mobile/src/components/OfferDetailScreen/index.tsx

```diff
@@ -1,52 +1,20 @@
 import {type RootStackScreenProps} from '../../navigationTypes'
 import Screen from '../Screen'
-import useSafeGoBack from '../../utils/useSafeGoBack'
-import {useTranslation} from '../../utils/localization/I18nProvider'
-import {useSingleOffer} from '../../state/marketplace'
-import {Text, YStack} from 'tamagui'
 import OfferInfo from './components/OfferInfo'
-import {isSome} from 'fp-ts/Option'
-import Button from '../Button'
 import KeyboardAvoidingView from '../KeyboardAvoidingView'
 
 type Props = RootStackScreenProps<'OfferDetail'>
 
 function OfferDetailScreen({
   route: {
-    params: {offerId},
+    params: {offer},
```

Comment:
https://reactnavigation.org/docs/params/#what-should-be-in-params

> Avoid passing the full data which will be displayed on the screen itself (e.g. pass a user id instead of user object).